rust_src/src/windows.rs

```diff
@@ -2034,3 +2035,15 @@ pub fn window_pixel_height_before_size_change(window: LispWindowValidOrSelected)
 }
 
 include!(concat!(env!("OUT_DIR"), "/windows_exports.rs"));
+
+/// Run `window-configuration-change-hook' for FRAME.
+/// If FRAME is omitted or nil, it defaults to the selected frame.
+#[lisp_fn(min = "0")]
+pub fn run_window_configuration_change_hook(frame: Option<LispObject>) -> LispObject {
+    let f = frame.map_or(Qnil, |obj| obj);
+    unsafe { run_window_conf_change_hook(decode_live_frame(f)) };
+    Qnil
```

Comment:
No need for the naked Qnil return. The `lisp_fn` wrapper handles this in the cases where the function should always return Qnil.